lib/embulk/input/google_adwords.rb

```diff
@@ -89,7 +89,7 @@ def run
         query << " DURING #{task["daterange"]["min"]},#{task["daterange"]["max"]}" unless task["daterange"].empty?
 
         query_report_results(query) do |row|
-          next if row.empty?
+          next if row.blank?
```

Comment:
ActiveSupport入ってなさそうなので

```suggestion
          next if row.nil? || row.empty?
```

でしょうか

Reply:
（では、なんで動いているんだこいつは・・・）

Reply:
ローカルのembulkとかには入っているのかなと